app/controllers/twilio_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'twilio-ruby'
+
+class TwilioController < ApplicationController
+  include Webhookable
+
+  after_filter :set_header
+
+  skip_before_action :verify_authenticity_token
+
+  def connect_customer
+    # mocking a db of customers, connect to real db in production
+    customers = {
+      '123' => {'phone_number' => '7724183162'},
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.